python/jupyterlab_widgets/src/manager.ts

```diff
@@ -455,7 +455,7 @@ export class WidgetManager extends LabWidgetManager {
    */
   private _saveState(): void {
     const state = this.get_state_sync({ drop_defaults: true });
-    this._context.model.metadata.set('widgets', {
+    this._context.model.setMetadata('widgets', {
```

Comment:
This is Lab 4 specific. So it should be tested like

```
if ( this._context.model.setMetadata ) {
} else {
}
```

Reply:
Thanks!